lib/framework/connector_settings.rb

```diff
@@ -12,16 +12,18 @@ module Framework
   class ConnectorSettings
     def self.fetch(connector_package_id)
       es_response = ElasticConnectorActions.load_connector_settings(connector_package_id)
-
+      if es_response.nil? || es_response['found'] == false
+        nil
+      end
```

Comment:
I think it's great to log here the fact that nothing was found, plus what's the case when `es_response.nil?` is true? Is it even possible?

Reply:
I'm not so well acquainted with the ruby lib, so IDK if there's a nil possibility there. I think I'm just checking for it "in case". We can probably cut this.
I don't really want to log the not found case, since it might be quite common that if you register a new connector, the old settings aren't found. I don't want it to look like an error condition.